File: Server/src/main/ua/nure/easygo/model/Child.java
package ua.nure.easygo.model;

import java.util.Calendar;

/**
 * Created by Oleg on 29.10.2016.
 */
public class Child {
    private String name;
    private Calendar birthDate;

    public Child(String name, Calendar birthDate) {
        this.name = name;
        this.birthDate = birthDate;
    }

    public Child() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Calendar getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(Calendar birthDate) {
        this.birthDate = birthDate;
    }
}

File: Client/app/src/main/java/ua/nure/havrysh/kindergarten/model/Child.kt
package ua.nure.havrysh.kindergarten.model

import java.sql.Date

data class Child(
    var id: String = "",
    var name: String? = null,
    var surname: String? = null,
    var birth_date: Date? = null,
    var group_id: Long = 0,
    var parent: Human = Human(),
    var notice: String? = null
) {
    
    fun getFullName() = "$name $surname"
    
    val icon: String
        get() = "child_$id"
}